src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -201,6 +205,43 @@ public static boolean setupDependencies(ProjectConfiguration configuration) thro
         return true;
     }
 
+    /**
+     * Returns the path to the llc compiler that is working for the provided configuration.
+     * The <code>configuration</code> object must have its host triplet set correctly.
+     * If the llc compiler is found in the file cache, it will be returned. Otherwise, it will
+     * be downloaded and stored in the cache. After calling this method, it is guaranteed that an
+     * llc compiler is in the specified path.
+     * <p>
+     *     There might be different versions for the llc compiler, but this is handled inside
+     *     Substrate. If the developer wants a specific flavour of llc, it is recommended to
+     *     use <code>configuration.setLlcPath()</code> which takes precedence over calling this method.
+     * </p>
+     * @param configuration
+     * @return the path to a working llc compiler.
+     * @throws IOException in case the required directories can't be created or navigated into.
+     */
+    public static Path getLlcPath(ProjectConfiguration configuration) throws IOException {
+        Path llcRootPath = Constants.USER_SUBSTRATE_PATH.resolve("llc");
```

Comment:
Move `"llc"` to constants, like `Constants.LLC_NAME`

Reply:
ok